addons/website_crm_livechat/models/chatbot_script_step.py

```diff
@@ -0,0 +1,16 @@
+
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, models
+
+
+class ChatbotScriptStep(models.Model):
+    _inherit = 'chatbot.script.step'
+
+    def _chatbot_crm_prepare_lead_values(self, mail_channel, description):
+        values = super()._chatbot_crm_prepare_lead_values(mail_channel, description)
+        if mail_channel.livechat_visitor_id:
```

Comment:
Shouldn't we also try to get visitor phone, mobile, email, country, ... if not already defined on the customer / lead ?